sklearn/utils/estimator_checks.py

```diff
@@ -337,20 +355,21 @@ def _is_32bit():
     return struct.calcsize('P') * 8 == 32
 
 
-def check_estimator_sparse_data(name, Estimator):
+def check_estimator_sparse_data(name, estimator):
     rng = np.random.RandomState(0)
     X = rng.rand(40, 10)
     X[X < .8] = 0
     X_csr = sparse.csr_matrix(X)
     y = (4 * rng.rand(40)).astype(np.int)
+    y = multioutput_estimator_convert_y_2d(estimator, y)
```

Comment:
yeah I thought we deprecated that behavior when we introduced MaxAbsScaler but that was as an alternative for MinMaxScaler, not StandardScaler, my bad